ntoskrnl/ke/bug.c

```diff
@@ -715,8 +715,8 @@ KiDisplayBlueScreen(IN ULONG MessageId,
 
     /* Show the technical Data */
     sprintf(AnsiName,
-            "\r\n\r\n*** STOP: 0x%08lX (0x%p,0x%p,0x%p,0x%p)\r\n\r\n",
-            KiBugCheckData[0],
+            "\r\n\r\n*** STOP: 0x%p (0x%p,0x%p,0x%p,0x%p)\r\n\r\n",
```

Comment:
That change looks rather suspicious. A bug check code should remain on 8 digits. Be it 64 bits or 32 bits.